tests/diag.sh

```diff
@@ -2325,6 +2325,27 @@ snmp_stop_trapreceiver() {
     fi
 }
 
+omfwd_sni_server() {
+	# Args: 1=library, 2=port
+	# 1 is either "openssl" or "gnutls"
+	# 2 is the port the SNI server is running on
+	"./$1_sni_server" "$2" "$srcdir/tls-certs/cert.pem" "$srcdir/tls-certs/key.pem" 1>"$RSYSLOG_DYNNAME.sni-server.stdout" &
+	echo "$!" >"$RSYSLOG_DYNNAME.sni-server.pid"
+}
+
+omfwd_sni_check() {
+    # Args: 1=sni
+	# 1 is the SNI we're expecting (can be omitted to confirm no SNI is sent)
+	sni=${1:-""}
+
+	sleep 5
```

Comment:
sleep should be avoided except absolutely necessary - CI can be very slow and using fixed timing almost always leads to false positives

Reply:
Ah, okay. Sorry for the delay - holidays etc. - I'm still working on figuring out why the tests are failing on certain distros.

Reply:
I am currently working on the CI. Due to this, there is a higher rate of flakes than usual.  That said, you also need to rebase to current master. Otherwise we are no longer able to do CI runs (see merge conflicts below).